install/collecting_ncn_mac_addresses.md

```diff
@@ -50,7 +50,7 @@ For help with either of those, see [LiveCD Setup](bootstrap_livecd_remote_iso.md
     ```bash
     pit# mv /var/www/boot/script.ipxe /var/www/boot/script.ipxe.bak
     ```
-2. Verify consoles are active with `conman -q`,
+2. Verify consoles are active with `conman -q` - this lists all of the nodes that conman is connected to,
```

Comment:
Change to:

2. Verify consoles are active with `conman -q`.

The following command lists all nodes connected to ConMan.

Reply:
I struggled a little with the wording on this.  The list is not connected nodes, but the ones it is 'trying to connect to', or 'configured to connect to'.  It is entire possible to get a node on the list, call 'conman -j xname' and get a reply 'Not connected to xname'.  I don't know how confused people would be about that...